content/_code-samples/markers-and-pagination/py/pagination-with-markers.py

```diff
@@ -0,0 +1,23 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests import LedgerData
+
+# Create a client to connect to the network.
+client = JsonRpcClient("https://xrplcluster.com/")
+
+# Specify a ledger to query for info.
+ledger = LedgerData(ledger_index=500000)
```

Comment:
Same as above, the best practice we want to showcase is looking up by `"validated"` the first time and then specifying the ledger_index from the response in each follow-up request.